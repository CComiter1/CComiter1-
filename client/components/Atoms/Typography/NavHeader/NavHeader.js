import styled from 'styled-components'

export default styled.h1`
  color: ${props => props.theme.accentColor};
  font-size: 2.3rem;
  margin: 0;
  text-transform: uppercase;
  font-family: 'Roboto', sans-serif;
`
