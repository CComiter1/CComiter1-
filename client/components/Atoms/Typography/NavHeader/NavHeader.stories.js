import React from 'react'
import { storiesOf } from '@storybook/react'

import NavHeader from './NavHeader'

storiesOf('Atoms/Typography', module)
  .add('NavHeader', () => <NavHeader>Accenture Call Center</NavHeader>)
