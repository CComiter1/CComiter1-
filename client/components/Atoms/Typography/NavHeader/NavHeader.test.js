import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import NavHeader from './NavHeader'

describe('NavHeader', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <NavHeader>NAV HEADER TEXT</NavHeader>
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
