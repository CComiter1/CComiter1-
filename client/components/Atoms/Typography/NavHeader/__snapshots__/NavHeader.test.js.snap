// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavHeader renders correctly 1`] = `
.c0 {
  font-size: 2.3rem;
  margin: 0;
  text-transform: uppercase;
  font-family: 'Roboto',sans-serif;
}

<h1
  className="c0"
>
  NAV HEADER TEXT
</h1>
`;
